lib/experimental/Information/StatusTag/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { useTextFormatEnforcer } from "@/lib/text"
+import { cn } from "@/lib/utils"
+import { forwardRef } from "react"
+
+type Variant = "neutral" | "info" | "positive" | "warning" | "critical"
+
+interface Props {
+  text: string
+  variant: Variant
+}
+
+export const StatusTag = forwardRef<HTMLDivElement, Props>(
+  ({ text, variant }, ref) => {
+    useTextFormatEnforcer(text, { disallowEmpty: true })
+
+    return (
+      <div
+        ref={ref}
+        className={cn(
+          "flex flex-row items-center justify-start gap-1.5 rounded-md px-2 py-0.5 text-base font-medium",
+          {
+            neutral: "bg-f1-background-secondary text-f1-foreground-secondary",
+            info: "bg-f1-background-info text-f1-foreground-info",
+            positive: "bg-f1-background-positive text-f1-foreground-positive",
+            warning: "bg-f1-background-warning text-f1-foreground-warning",
+            critical: "bg-f1-background-critical text-f1-foreground-critical",
+          }[variant]
+        )}
+      >
+        <div
```

Comment:
can we use our internal `Icon` component here?

Reply:
Not really, it's just a dot, I don't see the point of using a SVG here 🤔